app/core/hooks/useSearch.ts

```diff
@@ -9,9 +9,10 @@ type R = [zng.Record[], boolean]
 export default function useSearch(query: string, deps?: any[]): R {
   const dispatch = useDispatch<AppDispatch>()
   const [records, setRecords] = useState<zng.Record[]>([])
-  const [isFetching, setIsFetching] = useState<boolean>(false)
+  const [isFetching, setIsFetching] = useState<boolean>(true)
 
   useEffect(() => {
+    setIsFetching(true)
```

Comment:
if we default `isFetching` to `true` in the useState above, can we not rely on the response.status handler below to solely maintain the fetching state?

Reply:
The status handler takes a few milliseconds to update the response which resulted in a flash of empty state before isFetching ui kicked in.

Setting it here early fixed it.